sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -110,29 +163,83 @@ class PrecisionRecallDisplay(_BinaryClassifierCurveDisplayMixin):
 
     def __init__(
         self,
-        precision,
-        recall,
+        precisions,
+        recalls,
         *,
-        average_precision=None,
-        estimator_name=None,
+        average_precisions=None,
+        names=None,
         pos_label=None,
         prevalence_pos_label=None,
+        precision="deprecated",
+        recall="deprecated",
+        average_precision="deprecated",
+        name="deprecated",
     ):
-        self.estimator_name = estimator_name
-        self.precision = precision
-        self.recall = recall
-        self.average_precision = average_precision
+        self.precisions = _deprecate_singular(precision, precisions, "precision")
+        self.recalls = _deprecate_singular(recall, recalls, "recall")
+        self.average_precisions = _deprecate_singular(
+            average_precision, average_precisions, "average_precision"
+        )
+        self.names = _deprecate_singular(name, names, "name")
         self.pos_label = pos_label
         self.prevalence_pos_label = prevalence_pos_label
 
+    def _get_line_kwargs(self, n_multi, names, fold_line_kws, **kwargs):
+        """Get validated line kwargs for each curve."""
+        # Ensure parameters are of the correct length
+        names_ = [None] * n_multi if names is None else names
+        average_precisions_ = (
+            [None] * n_multi
+            if self.average_precisions is None
+            else self.average_precisions
+        )
+        # `fold_line_kws` ignored for single curve plots
+        # `kwargs` ignored for multi-curve plots
+        if n_multi == 1:
+            fold_line_kws = [kwargs]
+        else:
+            if fold_line_kws is None:
+                fold_line_kws = [
+                    {"alpha": 0.5, "color": "tab:blue", "linestyle": "--"}
```

Comment:
If we make all multi curve lines the same appearance, the legend will not be relevant.
Maybe we should not specify color?

![image](https://github.com/user-attachments/assets/4391a383-2440-4f2a-a4f9-1e4a065ff7f4)